site/content/usage/11-customizing-the-kubelet.md

```diff
@@ -53,7 +53,8 @@ nodeGroups:
 In this example, given instances of type `m5a.xlarge` which have 4 vCPUs and 16GiB of memory, the `Allocatable` amount
 of CPUs would be 3.4 and 15.4 GiB of memory. In addition, the `DynamicKubeletConfig` feature gate is also enabled. It is
 important to know that the values specified in the config file for the the fields in `kubeletExtraconfig` will 
-completely overwrite the default values specified by eksctl.
+completely overwrite the default values specified by eksctl. However, omitting one or more `kubeReserved` parameters
+will cause the missing parameters to be defaulted to sane values based on the aws instance type being used.
```

Comment:
@rndstr It would be amazing if there was a chart of these for the most popular instances right in the docs instead of having to look at the code. At the very least, possibly link to the part in the Code where these are defined? Also, you should certainly mention instance types which aren't supported by this.

Reply:
Hi @InAnimaTe , thanks, I think this is a good idea. I created the following issue https://github.com/weaveworks/eksctl/issues/2091 to track this